Formula/j/julia.rb

```diff
@@ -197,6 +194,8 @@ def install
 
     # Create copies of the necessary gcc libraries in `buildpath/"usr/lib"`
     system "make", "-C", "deps", "USE_SYSTEM_CSL=1", "install-csl"
+    return if OS.linux? && Hardware::CPU.arm?
```

Comment:
Same failure so something in this particular `make` is misbehaving in a way that causes Ruby to not like result.